### ISSUE TITLE
[Request/question] Extract Lora from Checkpoint

### ISSUE THREAD

**Issue by Omenizer** [Linaqruf/kohya-trainer#89]
Hi, I know extraction is possible with the Kohya utils, but is there a way to do this in colab? Can't run locally :(

Thanks! (and sorry for posting this under issues)

**Comment by GeneralAwareness** [comment]
yeah, I always feel bad myself as it isn't really an issue I am posting about, but there is no where else to post it (a github limitation that a community tab doesn't help when offered).

I spent a month trying to figure out why I could not train a Lora (or a delta) on colab or locally and come to find out it was because it, nor my gpu, has bf16 and I was using 128.  OOPS.  100 seems to be the fp16 max as the more I go the weaker the results until 128 dim/alpha gives me nothing.  Now the reason I am bringing this up here is that I tried the same on extraction and the same exact results on my model.  I barely could see it.  I suspect BF16 is needed for it too.

**Comment by Linaqruf** [comment]
I'll write that next time, but not now, thanks for the suggestion!
